tools/import/index.html

```diff
@@ -55,8 +55,8 @@ <h1>Bulk Import</h1>
                             <input id="timeout-input" type="number" value="10000" min="0" step="1000">
                         </div>
                         <div>
-                            <label for="enableJavaScript">Enable JavaScript</label>
-                            <input type="checkbox" id="enableJavaScript" name="enableJavaScript">
+                            <label for="enableJavascript">Enable JavaScript</label>
```

Comment:
Should we change the label too?

```suggestion
                            <label for="enableJavascript">Enable Javascript</label>
```